src/mlpack/methods/neighbor_search/ns_model_impl.hpp

```diff
@@ -464,6 +498,8 @@ std::string NSModel<SortPolicy>::TreeName() const
       return "R tree";
     case R_STAR_TREE:
       return "R* tree";
+    case X_TREE:
+      return "X tree";
```

Comment:
If you don't mind, when I merge this I'll probably just change all of the x-tree cases to come after the ball tree, like in the enum.  It's a really trivial change but it preserves the ordering and makes it easier to read, in my opinion.